.github/workflows/build_frameworks.yml

```diff
@@ -93,6 +93,15 @@ jobs:
           git config --local user.name ${{ github.actor }}
           git add .
           git commit -m "Update podspec repo tag:${{ github.event.release.tag_name }}"
-          git push 
+          git push
         env:
-          API_TOKEN_GITHUB: ${{ secrets.RIVE_REPO_PAT }}
\ No newline at end of file
+          API_TOKEN_GITHUB: ${{ secrets.RIVE_REPO_PAT }}
+        - name: Register trunk session
+          run: |
+            pod trunk register 'luigi@erive.app' 'Luigi' --description="Session no.${{ github.run_number }}"
```

Comment:
Can this not be done by providing the token via a secret or something? Also, does this really need to run with every build? Email address is incorrect here btw! (there's an extra e at the start of rive.app).

Reply:
AFAIK token is valid for 128 days so I think that generating it for every build is just more convenient :). I'll fix email address ofc :D

Reply:
Sorry, I misunderstood how this token verification works :). Totally agree with you guys!